server.js

```diff
@@ -12,6 +13,8 @@ server.set('views', path.join(__dirname, './', 'views'));
 
 server.use(express.static(path.join(__dirname, './', 'public')));
 server.use(router);
+server.use(bodyParser.urlencoded({ extended: true }));
+server.use(bodyParser.json());
```

Comment:
does the order of these `server.use(...` lines matter?

Reply:
No, but for orderly